packages/tailwindcss-react-aria-components/src/index.js

```diff
@@ -5,37 +5,60 @@ const attributes = {
     ['hover', 'hovered'],
     ['focus', 'focused'],
     'focus-visible',
+    'focus-within',
     'pressed',
     'disabled',
     'drop-target',
     'dragging',
     'empty',
+    'allows-dragging',
     'allows-removing',
     'allows-sorting',
     'placeholder',
     'selected',
     'indeterminate',
-    'readonly',
+    ['read-only', 'readonly'],
     'required',
     'entering',
     'exiting',
     'open',
     'unavailable',
+    'outside-month',
+    'outside-visible-range',
+    'selection-start',
+    'selection-end',
     'current',
-    'invalid'
+    'invalid',
+    'resizing'
   ],
   enum: {
     placement: ['left', 'right', 'top', 'bottom'],
     type: ['literal', 'year', 'month', 'day'],
     layout: ['grid', 'stack'],
-    orientation: ['horizontal', 'vertical']
+    orientation: ['horizontal', 'vertical'],
+    'selection-mode': ['single', 'multiple'],
+    'resizable-direction': ['right', 'left', 'both'],
+    'sort-direction': ['ascending', 'descending']
   }
 };
 
+const shortNames = {
+  'selection-mode': 'selection',
+  'resizable-direction': 'resizable',
+  'sort-direction': 'sort'
+};
```

Comment:
Writing `resizeable-direction-right:` seemed a bit long, so this lets you write `resizable-right:`. Bad idea?

Reply:
All the shortened ones look good to me.